widget-context.php

```diff
@@ -418,16 +418,10 @@ function context_check_url( $check, $settings ) {
 
 	function match_path( $patterns ) {
 
-		global $wp;
-
 		$patterns_safe = array();
 
-		// Get the request URI from WP
-		$url_request = $wp->request;
-
-		// Append the query string
-		if ( ! empty( $_SERVER['QUERY_STRING'] ) )
-			$url_request .= '?' . $_SERVER['QUERY_STRING'];
+		// Get the request URI
+		$url_request = trim( $_SERVER['REQUEST_URI'], '/' );

```

Comment:
@lachmanski I'm not sure. I personally think that your approach is better in terms of user experience and expectations. We just need to be careful about not breaking existing configurations.

Do you know if `$wp->request` is relative to WP or the root?